test/test_index.py

```diff
@@ -41,3 +41,34 @@ def test_get_index_v3_invalid():
         assert False
     except AssertionError:
         pass
+
+
+def test_get_index_from_http_with_query_parameters():
+    import time
```

Comment:
Please move this line two lines lower (below `sys`) to keep the import order alphabetically.